apps/frontend/src/components/instrument/CreateUpdateInstrument.tsx

```diff
@@ -72,7 +72,7 @@ const CreateUpdateInstrument = ({
 
     try {
       await api()
-        .getUsers({ filter: value })
+        .getUsers({ filter: value, userRole: UserRole.INSTRUMENT_SCIENTIST })
         .then((data) => {
           if (data.users?.totalCount == 0) {
             setFieldError('surname', 'No users found with that surname');
```

Comment:
Could this error message be updated to reflect the filter change? There's a similar error message in the other updated file that I think should be updated too.
```suggestion
            setFieldError('surname', 'No instrument scientists found with that surname');
```

Reply:
Actually, I think this might need to use the `instrumentSci` field from the translation file to be accurate at each facility.